google/cloud/pubsub/options.h

```diff
@@ -220,12 +220,38 @@ struct CompressionAlgorithmOption {
   using Type = int;
 };
 
+/**
+ * The maximum number of Open Telemetry span links.
+ *
+ * This option controls the number of links contained in a single publish span.
+ *
+ * @note Application developers should keep in mind that Cloud Trace
+ *    sets [limits][cloud-trace-quota-link] on the number of links per span
```

Comment:
Indentation?

Reply:
I took it from what I saw 

```
/**
 * The maximum deadline for each incoming message.
 *
 * Configure how long the application has to respond (ACK or NACK) to an
 * incoming message. Note that this might be longer, or shorter, than the
 * deadline configured in the server-side subscription.
 *
 * The value `0` is reserved to leave the deadline unmodified and just use the
 * server-side configuration.
 *
 * @note The deadline applies to each message as it is delivered to the
 *     application, thus, if the library receives a batch of N messages their
 *     deadline for all the messages is extended repeatedly. Only once the
 *     message is delivered to a callback does the deadline become immutable.
 *
 * @ingroup pubsub-options
 */
struct MaxDeadlineTimeOption {
  using Type = std::chrono::seconds;
};
```

I'm not really sure what is preferred

Reply:
Nevermind i see what you mean now

Reply:
done